lib/util/cache/package/decorator.ts

```diff
@@ -66,21 +70,59 @@ export function cache<T>({
       return callback();
     }
 
-    const cachedResult = await packageCache.get<unknown>(
+    finalKey = `cache-decorator:${finalKey}`;
+    const oldRecord = await packageCache.get<DecoratorCachedRecord>(
       finalNamespace,
       finalKey
     );
 
-    if (cachedResult !== undefined) {
-      return cachedResult;
+    const softTTL = ttlMinutes;
+    const cacheHardTtlMinutes = GlobalConfig.get().cacheHardTtlMinutes ?? 0;
+    let hardTTL = softTTL;
+    hardTTL +=
+      methodName === 'getReleases' || methodName === 'getDigest'
+        ? cacheHardTtlMinutes
+        : 0;
```

Comment:
```suggestion
    if (methodName === 'getReleases' || methodName === 'getDigest') {
      hardTtl = Math.max(cacheHardTtlMinutes, softTtl);
    }
```

I don't think we should add them?